### PR TITLE
images have standard height when uploaded

### PR DESCRIPTION
**Pull request by itsalwaysbenny**
added a standard image height to images when they are uploaded, this will keep the aspect ratio correct.

### CODE REVIEW

**Review by baree99 — DISMISSED** [review]
Please resolve conflicts

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <% @posts.order(created_at: :desc).each do |post| %>
+
   <div id="post">
     <div id="post-body">
-      <%= image_tag post.image_url.to_s %>
+      <% if post.image_url %>
```
Comment (will-gant) [inline app/views/posts/index.html.erb]: This seems to display exactly the same way if I comment out lines 5 and 7 (I tried this with a mixture of posts that had only text and posts that had only images).

I suspect that line 6 already only runs if post.image_url is true, and that therefore the 'if' statement is redundant...

**Review by will-gant — CHANGES_REQUESTED** [review]
See comment for your changes in posts/index.html.erb

**Review by will-gant — APPROVED** [review]

**Comment by will-gant** [comment]
Great. This ready for @neoeno now.

**Review by neoeno — APPROVED** [review]
Looks good! 🚢

**Pull request merged**